exporter-adapters/src/test/groovy/io/opentelemetry/auto/exporteradapters/ExporterAdaptersTest.groovy

```diff
@@ -0,0 +1,34 @@
+import io.opentelemetry.auto.exportersupport.SpanExporterFactory
+import io.opentelemetry.auto.tooling.ExporterClassLoader
+import spock.lang.Shared
+import spock.lang.Specification
+
+class ExporterAdaptersTest extends Specification {
+  @Shared
+  def projectVersion = System.getProperty("projectVersion")
+  def adapterRoot = System.getProperty("adapterRoot")
+
+  def "test exporter load"() {
+    setup:
+    def file = new File("${adapterRoot}/${exporter}-adapter/build/libs/${exporter}-adapter-${projectVersion}-all.jar")
+    println "Attempting to load ${file.toString()} for ${classname}"
+    assert file.exists(): "${file.toString()} does not exist"
+    URL[] urls = [file.toURI().toURL()]
+    def cl = new ExporterClassLoader(urls, this.getClass().getClassLoader())
+    def sl = ServiceLoader.load(SpanExporterFactory, cl)
```

Comment:
i'm personally a fan of longer variable names (and especially when there's no type name)